### PR TITLE
Feat: Add Phantom provider to support Solana

### PR DESCRIPTION
**Pull request by gmolki**

### CODE REVIEW

**Comment by gmolki** [comment]
### Connecting and switching to solana wallet demo

https://github.com/user-attachments/assets/dd8a07e9-5878-42b8-9224-fb55cee53cf2

File: src/components/pages/computing/NewInstancePage/cmp.tsx [hunk 1]
```diff
@@ -64,6 +64,17 @@ export default function NewInstancePage({ mainRef }: PageProps) {
 
   // ------------------
 
+  const isBlockchainHoldingCompatible = useCallback(
```
Comment (philogicae) [inline src/components/pages/computing/NewInstancePage/cmp.tsx:67]: We should move it to SDK later (in evm_utils we have similar ones)

File: src/domain/connect/base.ts [hunk 2]
```diff
@@ -48,15 +50,27 @@ export const providers: Record<ProviderId, Provider> = {
...
+} as Record<BlockchainId, ProviderId>
+
 // ------------------------
 
 export type Blockchain = {
   id: BlockchainId
   name: string
   chainId: number
   eip155: boolean
+  solana: boolean
```
Comment (philogicae) [inline src/domain/connect/base.ts:73]: This new field is not needed, we can use eip155: false https://namespaces.chainagnostic.org/eip155/caip2

File: src/hooks/pages/solutions/manage/useManageInstance.ts [hunk 3]
```diff
@@ -80,6 +80,15 @@ export function useManageInstance(): ManageInstance {
     getMapped()
   }, [sshKeyManager, instance])
 
+  const isBlockchainHoldingCompatible = useCallback(
```
Comment (philogicae) [inline src/hooks/pages/solutions/manage/useManageInstance.ts:83]: To move it to SDK later

**Review by philogicae — APPROVED** [review]

File: src/components/pages/computing/NewInstancePage/cmp.tsx (hunk 1 above)
Comment (gmolki) [reply src/components/pages/computing/NewInstancePage/cmp.tsx]: I moved it to a domain file to be reusable, so we can handle the logic easier in the console page without needing to change to the sdk as it's not needed in another application by now.

File: src/domain/connect/base.ts (hunk 2 above)
Comment (gmolki) [reply src/domain/connect/base.ts:73]: I thought on adding it as it is not harmful and will be easier to add new chains (e.g. polkadot) easily in the future

**Pull request merged**